packages/FAQ/docs/view-existing-invoice.md

```diff
@@ -0,0 +1,19 @@
+---
+title: View Existing Invoice 
+---
+
+## Step 1: Navigate Existing Invoice(s)
```

Comment:
This section seems more focused on how to use Smart Invoice as a client. If that's the intent, then I suggest titling the page that way. "viewing existing invoices" is a subtask a client does *in service of* the job of depositing or releasing. They will be looking for help on the job they're trying to complete rather than a specific subtask.

Also, let's acknowledge that a client may have received a direct link to an invoice rather than finding it via search.

Reply:
Really awesome point.